src/Compilers/CSharp/Test/Emit/PDB/PDBConstantTests.cs

```diff
@@ -14,7 +15,244 @@ public class PDBConstantTests : CSharpTestBase
         private CultureInfo _testCulture = new CultureInfo("en-US");
 
         [Fact]
-        public void TestSimpleLocalConstants()
+        public void StringsWithSurrogateChar()
```

Comment:
Seems like those tests are unrelated to this PR (CDI refactoring for dynamic and tuple names).